sql/src/main/java/io/crate/analyze/AnalyzedColumnDefinition.java

```diff
@@ -329,4 +335,23 @@ public void isParentColumn(boolean isParentColumn) {
     public boolean isParentColumn() {
         return isParentColumn;
     }
+
+    public void formattedGeneratedExpression(String formattedGeneratedExpression) {
+        this.formattedGeneratedExpression = formattedGeneratedExpression;
+    }
+
+    @Nullable
+    public String formattedGeneratedExpression() {
+        return formattedGeneratedExpression;
+    }
+
+    public void generatedExpression(Expression generatedExpression) {
+        this.generatedExpression = generatedExpression;
+    }
+
+    @Nullable
+    public Expression generatedExpression() {
```

Comment:
can you add docs to explain the difference between formattedGeneratedExpression and just generatedExpression?